treeherder/webapp/api/intermittents_view.py

```diff
@@ -88,8 +88,7 @@ def list(self, request):
 
         for item in self.queryset:
             item['lines'] = grouped_lines.get(item['job_id'], [])
-
-            match = filter(lambda x: item['job__option_collection_hash'] == x, hash_list)
+            match = [item['job__option_collection_hash'] for x in hash_list if item['job__option_collection_hash'] == x]
```

Comment:
It seems like this could use `set()` instead and avoid the manual list comprehension entirely?
https://docs.python.org/2/library/stdtypes.html#set-types-set-frozenset

Reply:
heh, yeah I went for the set originally. I'll change it back.